src/rest/DependenciesCtrl.ts

```diff
@@ -0,0 +1,165 @@
+import restify = require('restify');
+
+export enum DependencyTypes {
+    Association = "association",
+    Inheritance = "inheritance",
+    Implementation = "implementation",
+    Dependency = "dependency",
+    Aggregation = "aggregation",
+    Composition = "composition",
+    References = "references",
+    Calls = "calls"
+}
+
+export default class DependenciesCtrl {
+    private server: restify.Server;
+
+    /**
+     * Initializes the dependencies server object
+     */
+    constructor() {
+        this.server = restify.createServer({
+            name: 'It-depends',
+            version: '1.0.0'
+        });
+    }
+
+    /**
+     * Starts server, listens on port 8080.
+     *
+     * Note: we can update return value in promise to something more useful, currently void.
+     * @returns {Promise<any>}
+     */
+    public async start(): Promise<any> {
+
+        console.log("STARTING UP SERVER: " + this.server.name);
+
+        this.server.use(restify.plugins.acceptParser(this.server.acceptable));
+        this.server.use(restify.plugins.queryParser());
+        this.server.use(restify.plugins.bodyParser());
+
```

Comment:
Any other types of data FE will want to display?

And what does everyone think of the endpoint format `/<graph-type>/<level>/:<startcommit SHA>..<endcommit SHA>`

Reply:
Sounds good to me, does restify have support for query parameters? Something like

```
GET /deps/:graphType/:level?start=<startSHA>&end=<endSHA>
```